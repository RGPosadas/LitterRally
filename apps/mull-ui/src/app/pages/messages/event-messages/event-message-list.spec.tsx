import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import EventMessageList from './event-message-list';

describe('EventMessageList', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<EventMessageList />);
    expect(baseElement).toBeTruthy();
  });
  it('should match snapshot', () => {
    const tree = renderer.create(<EventMessageList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
