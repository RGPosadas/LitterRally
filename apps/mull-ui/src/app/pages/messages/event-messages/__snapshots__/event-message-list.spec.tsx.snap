// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventMessageList should match snapshot 1`] = `
<div
  className="page-container"
>
  <div
    className="custom-text-input-container "
  >
    <label
      className="custom-text-input-label"
      htmlFor="searchField"
    />
    <input
      className="custom-text-input input-border "
      data-testid="custom-text-input"
      id="searchField"
      name="searchField"
      onChange={[Function]}
      placeholder="Search"
      type="text"
      value=""
    />
  </div>
  <div>
    <div>
      <div
        className="event-bullet-container"
      >
        <img
          alt="event"
          className="event-bullet-photo"
          src="http://localhost:3333/api/media/1"
        />
        <div
          className="event-bullet-date-box"
        >
          <span
            className="event-box-bold-font"
          >
            11
          </span>
          <span
            className="event-box-font"
          >
            Dec
          </span>
          <span
            className="event-box-font"
          >
            22
            :
            0
          </span>
        </div>
        <div>
          Test title
        </div>
      </div>
    </div>
    <div>
      <div
        className="event-bullet-container"
      >
        <img
          alt="event"
          className="event-bullet-photo"
          src="http://localhost:3333/api/media/2"
        />
        <div
          className="event-bullet-date-box"
        >
          <span
            className="event-box-bold-font"
          >
            11
          </span>
          <span
            className="event-box-font"
          >
            Dec
          </span>
          <span
            className="event-box-font"
          >
            22
            :
            0
          </span>
        </div>
        <div>
          Test title
        </div>
      </div>
    </div>
    <div>
      <div
        className="event-bullet-container"
      >
        <img
          alt="event"
          className="event-bullet-photo"
          src="http://localhost:3333/api/media/3"
        />
        <div
          className="event-bullet-date-box"
        >
          <span
            className="event-box-bold-font"
          >
            11
          </span>
          <span
            className="event-box-font"
          >
            Dec
          </span>
          <span
            className="event-box-font"
          >
            22
            :
            0
          </span>
        </div>
        <div>
          Test title
        </div>
      </div>
    </div>
  </div>
</div>
`;
